### PR TITLE
fix: change way of checking if profile is an OID to use regex

### PR DESCRIPTION
**Pull request by omrozowicz-splunk**

### CODE REVIEW

File: splunk_connect_for_snmp_poller/manager/task_utilities.py [hunk 1]
```diff
@@ -778,3 +779,14 @@ def build_contextData(version, community, server_config):
...
+
+def is_oid(profile: str) -> bool:
+    """
+    This function checks if profile is an OID. OID is defined as a string of format:
+        - ex. 1.3.6.1.2.1.2
+        - ex. 1.3.6.2.1.*
+    @param profile: variable from inventory file, can be a name of profile or an OID
+    @return: if the profile is of OID structure
+    """
+    return bool(re.match(r"^(\d+\.?)*\*?$", profile))
```
Comment (weliasz) [inline splunk_connect_for_snmp_poller/manager/task_utilities.py:792]: This pattern would match "232." and empty string. I've found sth more detailed in the net:
^([1-9][0-9]{0,3}|0)(\.([1-9][0-9]{0,3}|0)){5,13}$

File: splunk_connect_for_snmp_poller/manager/tasks.py [hunk 2]
```diff
@@ -19,11 +19,12 @@
 from celery.utils.log import get_task_logger
 from pysnmp.hlapi import ObjectIdentity, ObjectType, SnmpEngine
 
-from splunk_connect_for_snmp_poller.manager.celery_client import app
+from splunk_connect_for_snmp_poller.manager.celery_client import app  # noqa: E401
```
Comment (weliasz) [inline splunk_connect_for_snmp_poller/manager/tasks.py:22]: What is # noqa: E401?

**Review by weliasz — APPROVED** [review]

File: splunk_connect_for_snmp_poller/manager/task_utilities.py (hunk 1 above)
Comment (omrozowicz-splunk) [reply splunk_connect_for_snmp_poller/manager/task_utilities.py]: As discussed, we don't need something as detailed to distinguish OID from profile

**Pull request merged**